enhancements/installer/gcp-customer-provided-subnets.md

```diff
@@ -0,0 +1,258 @@
+---
+title: (GCP) Allow Customer-Provisioned Subnets
+authors:
+  - "@jstuever"
+reviewers:
+  - "@abhinavdahiya"
+  - "@sdodson"
+approvers:
+  - "@abhinavdahiya"
+  - "@sdodson"
+creation-date: 2019-08-12
+last-updated: 2019-10-21
+status: implemented
+superseded-by:
+  - "https://docs.google.com/document/d/1qUYchUnw81cZNyvMM1cIGfF87uUsA3sZRSyh75_-cO4"
```

Comment:
I don't think we want an enhancement that's been replaced by a non-public Google Doc.  Can we update this enhancement if necessary to make it the canonical reference and drop `superseded-by`?